sample/mpp-library/build.gradle.kts

```diff
@@ -21,5 +21,11 @@ setupFramework(
 
 dependencies {
     mppLibrary(Deps.Libs.MultiPlatform.kotlinStdLib)
-    mppLibrary(Deps.Libs.MultiPlatform.mokoPermissions)
+    mppLibrary(Deps.Libs.MultiPlatform.coroutines)
+
+    // Adds mokoPermissions from Bintray:
+    // mppLibrary(Deps.Libs.MultiPlatform.mokoPermissions)
+
+    // Adds mokoPermissions locally as module (for the sample purposes):
+    mppModule(MultiPlatformModule(":permissions"))
```

Comment:
please remove this change. for testing locally updates of library see https://github.com/icerockdev/moko-permissions#set-up-locally (deploy update to mavenLocal and all will be ok)

Reply:
Yep, forgot about mavenLocal.